kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -191,11 +191,18 @@ data:
 
     # The following settings are "undocumented", we don't recommend uncommenting them unless directed by Microsoft.
     # Configuration settings for the waittime for the network listeners to be available
-    # [agent_settings.network_listener_waittime] 
+    # [agent_settings.network_listener_waittime]
     #   tcp_port_25226 = 45                           # Port 25226 is used for telegraf to fluent-bit data in ReplicaSet
     #   tcp_port_25228 = 60                           # Port 25228 is used for telegraf to fluentd data
     #   tcp_port_25229 = 45                           # Port 25229 is used for telegraf to fluent-bit data in DaemonSet
 
+    # [agent_settings.mdsd_config]
```

Comment:
arent we going to hide all this behind a scale based configmap option?

Reply:
as we discussed, on top of your mdsd limits PR, will expose single config entry for high_scale_config. Plan to support this and also single config entry for high_scale_config. certain scenarios, specific mdsd config will be useful.